src/components/utils/DEDialog.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * A general purpose dialog
+ *
+ * @author sriram
+ *
+ */
+import React from "react";
+import { useTranslation } from "i18n";
+
+import { build } from "@cyverse-de/ui-lib";
+
+import ids from "./ids";
+
+import {
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    IconButton,
+    useMediaQuery,
+    useTheme,
+} from "@material-ui/core";
+
+import CloseIcon from "@material-ui/icons/Close";
+
+function DEDialog(props) {
+    const { baseId, open, title, onClose, content, actions } = props;
+    const { t } = useTranslation("common");
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down("sm"));
+    const dialogTitleId = build(baseId, ids.DIALOG.TITLE);
+    return (
+        <Dialog
+            open={open}
+            onClose={onClose}
+            maxWidth="sm"
+            fullWidth
```

Comment:
Not necessary for this PR, but might be nice to make `maxWidth` and `fullWidth` configurable with props but also have these as default values perhaps